### DISCUSSION TITLE
New 319 bios for 2022 G14

### DISCUSSION THREAD

**Discussion by Inaiso** [seerge/g-helper#1110] [General]
So I was just curious what this new bios update ( 319) for the 2022 G14 (GA402RJ) was for? I read on the Asus rog website it was for security and didn't really have any more information. Anyone updated to version 319 yet? Anything of note? 

![image](https://github.com/seerge/g-helper/assets/132625303/bb79722a-0f48-450c-9046-e5ecd4490a88)

**Comment by seerge** [comment]
They never write what's new in each bios :) so it's to to community to try / test / figure out ...

**Comment by Inaiso** [comment]
So I updated to 319 because windows update did it automatically, nothings changed from what I can see. No idea what security it even updated either. Its all a big mystery...
